p5js/ImageClassification_TransferLearning/sketch.js

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2018 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+/* ===
+ML5 Example
+Image Classifier with Transfer Learning example with p5.js
+=== */
+
+let classifier;
+let video;
+let loss;
+let dogImages = 0;
+let catImages = 0;
+
+// Default options for training
+const options = {
```

Comment:
I wonder if we rely on the defaults for our basic example and leave this out. Great to allow this possibility but a little scary looking. I guess now with transfer learning we have to specify the number of classes in advance?

Reply:
These are totally optional, no need to provide them (actually they are the same in the src code). But if you are using more classes and examples in your training you might need to tweak this values a bit.

The defaults expect 2 classes.

Reply:
I wonder if there is some clever way we could wait to configure the number of new classes in the model until right before training.

This is on the borderline of perhaps too much magic behind the scenes, but having a flexible "teachable machine" where the interaction is making up classes and adding training images on the fly might be a useful feature.

Reply:
yes, totally. we could have predefined values that are updated based on the number of classes and images. Values we can get running a couple of examples and seeing performance.